x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/traces/critical_path.spec.ts

```diff
@@ -34,78 +34,86 @@ export default function ApiTest({ getService }: FtrProviderContext) {
   }
 
   // format tree in somewhat concise format for easier testing
```

Comment:
Is this comment supposed to be in this line? 
It seems we could move it to line `40`, where `formatTree` is.